mailbox/api/src/main/java/org/apache/james/mailbox/model/ParsedAttachment.java

```diff
@@ -35,7 +31,7 @@
 
         @FunctionalInterface
         interface RequireContent {
-            RequireName content(InputStream stream);
+            RequireName content(byte[] bytes);
```

Comment:
the pr is named "Remove byte array from attachment " and here we put it back, apparently.
Did I missed something?

Reply:
It is not an attachment stored in the mailbox-api but a parsed implementation of it.

Reply:
that means that parsing is no more streamed, while storing is now streamed?